gruntfile.js

```diff
@@ -358,27 +356,16 @@ module.exports = function (grunt) {
         dest: '<%= distdir %>/js/angular.js'
       }
     },
-    recess: { // TODO: not maintained, unable to preserve license comments, switch out for something better.
+    postcss: {
       build: {
-        files: {
-          '<%= distdir %>/css/<%= pkg.name %>.css': ['<%= src.css %>'],
-          '<%= distdir %>/css/vendor.css': ['<%= src.cssVendor %>']
-        },
         options: {
-          compile: true,
-          noOverqualifying: false // TODO: Added because of .nav class, rename
-        }
-      },
-      min: {
-        files: {
-          '<%= distdir %>/css/<%= pkg.name %>.css': ['<%= src.css %>'],
-          '<%= distdir %>/css/vendor.css': ['<%= src.cssVendor %>']
+          processors: [
+            autoprefixer({browsers: 'last 2 versions'}), // add vendor prefixes
+            cssnano() // minify the result
+          ]
         },
-        options: {
-          compile: true,
-          compress: true,
-          noOverqualifying: false // TODO: Added because of .nav class, rename
-        }
+        src: '<%= distdir %>/css/<%= pkg.name %>.css',
+        dest: '<%= distdir %>/css/<%= pkg.name %>.css'
```

Comment:
The `dest` value should be `'<%= distdir %>/css/app.css'` otherwise it gets overwritten.